docs/examples/ResultsReader.rst

```diff
@@ -0,0 +1,760 @@
+    univ3101.gc['betaEff'] # obtain beta-effective
+
+
+
+
+.. parsed-literal::
+
+    array([  3.04272000e-03,   8.93131000e-05,   6.59324000e-04,
+             5.62858000e-04,   1.04108000e-03,   5.67326000e-04,
+             1.22822000e-04])
+
+
+
+``Macro`` and ``Micro`` energy group structures are stored directly in
+the universe.
+
+ 
+
+    # Obtain the macro energy structure in MeV
+    univ3101.groups
+
+
+
+
+.. parsed-literal::
+
+    array([  1.00000000e+37,   1.00000000e+01,   6.06530000e+00,
+             3.67880000e+00,   2.23130000e+00,   1.35340000e+00,
+             8.20850000e-01,   4.97870000e-01,   3.01970000e-01,
+             1.83160000e-01,   1.11090000e-01,   6.73800000e-02,
+             4.08680000e-02,   2.47880000e-02,   1.50340000e-02,
+             9.11880000e-03,   5.53090000e-03,   3.35460000e-03,
+             2.03470000e-03,   1.23410000e-03,   7.48520000e-04,
+             4.54000000e-04,   3.12030000e-04,   1.48940000e-04,
+             0.00000000e+00])
+
+
+
+ 
+
+    # Obtain the micro energy structure in MeV
+    univ3101.microGroups[:5:] # print only the five first values
+
+
+
+
+.. parsed-literal::
+
+    array([  1.00000000e-10,   1.48940000e-04,   1.65250000e-04,
+             1.81560000e-04,   1.97870000e-04])
+
+
+
+Plotting universes
+------------------
+
+ 
+
+    # obtain the energy grid in descending order (high to low energy)
+    xdata = univ3101.groups[1:] 
+    # obtain the inifinite abs. xs
+    ydataInf = univ3101.infExp['infAbs']
+    ydataB1 = univ3101.b1Exp['b1Abs']
+
+ 
+
+    plt.plot(xdata, ydataInf,'r', label='INF')
+    plt.plot(xdata, ydataB1,'*g', label='B1')
+    plt.legend()
+    plt.xlabel('Energy, MeV'), plt.ylabel('Macroscopic absorption cross section, cm$^{-1}$')  
+
+
+
+
+.. parsed-literal::
+
+    (<matplotlib.text.Text at 0x216bc66d8d0>,
+     <matplotlib.text.Text at 0x216bc6a50f0>)
+
+
+
+
+.. image:: images/ResultsReader_51_1.png
+
+
+ 
+
+    # obtain the energy grid in descending order (high to low energy)
+    xdata = univ3101.groups[1:] 
+    # obtain the inifinite fiss. xs
+    ydata3101 = univ3101.infExp['infFiss'] # for universe 3101 and index=2
+    ydata3102 = univ3102.infExp['infFiss'] # for universe 3102 and index=4
+
+ 
+
+    plt.plot(xdata, ydata3101,'r', label='universe 3101')
+    plt.plot(xdata, ydata3102,'*g', label='universe 3102')
+    plt.legend()
+    plt.xlabel('Energy, MeV'), plt.ylabel('Macroscopic fission cross section, cm$^{-1}$')  
+
+
+
+
+.. parsed-literal::
+
+    (<matplotlib.text.Text at 0x216bc7a8e80>,
+     <matplotlib.text.Text at 0x216bc794940>)
+
+
+
+
+.. image:: images/ResultsReader_53_1.png
+
+
+User Defined Settings
+---------------------
+
+The user is able to filter the required information by using the
+settings option.
+
+A detailed description on how to use the settings can be found on:
+http://serpent-tools.readthedocs.io/en/latest/api/settings.html
```

Comment:
[Correct this in the notebook as well] Better link would be http://serpent-tools.readthedocs.io/en/latest/settingsTop.html, as this explains what all the settings are and how they influence the code